src/indexer.hh

```diff
@@ -322,13 +322,19 @@ struct IndexFile {
   std::string toString();
 };
 
+struct IndexResult {
+  std::vector<std::unique_ptr<IndexFile>> indexes;
+  int n_errs;
```

Comment:
specify default value =0, especially since you have `return {}` in the above function